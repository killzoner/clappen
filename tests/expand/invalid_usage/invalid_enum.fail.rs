fn main() {
    #[clappen::clappen(export = test)]
    mod m2 {
        pub enum Foo {}
    }
}
