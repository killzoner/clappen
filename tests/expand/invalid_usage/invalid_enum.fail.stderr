error: clappen support is limited to a single struct with one or more impl/use blocks
 --> tests/expand/invalid_usage/invalid_enum.fail.rs:2:24
  |
2 |     #[clappen::clappen(export = test)]
  |                        ^^^^^^^^^^^^^
